.idea/libraries/Maven__com_google_code_gson_gson_2_8_5.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```

Comment:
I don't think .idea files are in our .gitignore so probably best to not include this